management/connection.go

```diff
@@ -34,6 +34,7 @@ type Connection struct {
 	// character and can only contain alphanumeric characters and '-'. Max
 	// length 128.
 	Name *string `json:"name,omitempty"`
+	DisplayName *string `json:"display_name,omitempty"`
```

Comment:
Could you please run `go fmt` on this file?

Reply:
@alexkappa done!